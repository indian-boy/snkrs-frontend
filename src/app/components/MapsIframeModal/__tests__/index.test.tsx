import { render } from '@testing-library/react';
import * as React from 'react';
import { MapsIframeModal } from '..';

describe('<MapsIframeModal  />', () => {
  it('should match snapshot', () => {
    const mapsIframeModal = render(<MapsIframeModal />);
    expect(mapsIframeModal.container.firstChild).toMatchSnapshot();
  });
});
