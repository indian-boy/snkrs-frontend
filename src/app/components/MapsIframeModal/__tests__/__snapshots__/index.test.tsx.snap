// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MapsIframeModal  /> should match snapshot 1`] = `null`;
